lib/libzfs/libzfs_import.c

```diff
@@ -248,6 +248,17 @@ is_mpath_whole_disk(const char *path)
 	return (wholedisk);
 }
 
+static int
+udev_device_is_ready(struct udev_device *dev)
+{
+#ifdef HAVE_LIBUDEV_UDEV_DEVICE_GET_IS_INITIALIZED
+	return (udev_device_get_is_initialized(dev));
+#else
+	/* wait for DEVLINKS property to be initialized */
+	return (udev_device_get_property_value(dev, "DEVLINKS") != NULL);
```

Comment:
What would be the downside to using `udev_device_get_property_value()` even if `udev_device_get_is_initialized()` is available?  If there really isn't one it might make sense to always use that interface to keep the code simple.